src/TestInfrastructure/NoOpThreadHandler.cs

```diff
@@ -29,7 +29,7 @@ namespace SonarLint.VisualStudio.Integration.UnitTests
     /// </summary>
     /// <remarks>All operations are performed synchronously and return immediately.
     /// No thread switches take place.</remarks>
-    internal class NoOpThreadHandler : IThreadHandling
+    public class NoOpThreadHandler : IThreadHandling
```

Comment:
Made public so I could use it in the IssueViz.UnitTests project and replace IThreadHandling